app/models/lunch.rb

```diff
@@ -28,7 +32,11 @@ def einstein
         meals = menu.css('div.field-day').select{|day| valid_date?(week, day) }.flat_map do |day|
           day.css('p').to_a.reject{|m| invalid_meal?(m) }.map do |meal|
             content = meal.content.gsub(/[\s\u00A0]/, ' ').strip
-            tag_food(content) unless content.empty?
+            unless content.empty?
+              the_meal = tag_food(content)
+              the_meal[:allergens] = " - N/A"
```

Comment:
Then it would be represented the same way the meals with no data would which would be confusing